core/timeline_inc.php

```diff
@@ -44,14 +43,11 @@
 }
 
 echo '<div class="date-range">' . date( $t_short_date_format, $t_start_time ) . ' .. ' . date( $t_short_date_format, $t_end_time ) . $t_prev_link . $t_next_link . '</div>';
-$t_events = timeline_sort_events( $t_events );
 
-$t_num_events = timeline_print_events( $t_events, ( $f_all ? 0 : MAX_EVENTS ) );
+timeline_print_events( $t_events );
 
-# Don't display "More Events" link if there are no more entries to show
-# Note: as of 2015-01-19, this does not cover the case of entries excluded
-# by filtering (e.g. Status Change not in RESOLVED, CLOSED, REOPENED)
-if( !$f_all && $t_num_events < count( $t_events )) {
+# We will compromise accuracy of more link and showing exactly N without clicking more in favor or reducing load.
```

Comment:
Removal of the count here is due to the fact that the returned array me be less than 50, yet there is  more rows.  That is because we limit the number of rows returned from the database, then we do further filtering outside.  Unless we have a streaming implementation, we can't check the count here.  Also count is a best guess that there may be further rows, but is not guaranteed.  The best approach is to fetch count + 1, then check, which i wasn't doing anyway.